pajbot/bot.py

```diff
@@ -88,6 +88,12 @@ def __init__(self, config, args):
 
         if config["main"].getboolean("verified", False):
             TMI.promote_to_verified()
+        elif config["main"].getboolean("known", False):
+            TMI.promote_to_known()
```

Comment:
swap these, so `known = 1` and `verified = 1` lead to rate limits of a verified bot

```suggestion
        if config["main"].getboolean("known", False):
            TMI.promote_to_known()
        elif config["main"].getboolean("verified", False):
            TMI.promote_to_verified()
```

Reply:
This is still a problem, verified should take precedence over known

Reply:
I overlooked the elif you introduced, my bad